test-suite-aot/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'test-suite-aot'
```

Comment:
@melix do we need a `settings.gradle` for a submodule?

Reply:
No we don't. Unless it's an included build, which isn't the case here.